marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-clear-user-data.json

```diff
@@ -1,14 +1,19 @@
 {
   "role-name": "hub-central-clear-user-data",
-  "description": "This temporarily inherits data-hub-developer so it has the necessary privileges to use Data Hub and to insert artifacts. Will rework this once the new 5.3.0 roles are available.",
+  "description": "Permits clearing user data via Hub Central",
   "role": [
     "hub-central-user",
-    "data-hub-developer",
     "data-hub-entity-model-reader",
+    "data-hub-entity-model-writer",
     "data-hub-flow-reader",
+    "data-hub-flow-writer",
     "data-hub-ingestion-reader",
+    "data-hub-ingestion-writer",
     "data-hub-mapping-reader",
+    "data-hub-mapping-writer",
     "data-hub-match-merge-reader",
-    "data-hub-step-definition-reader"
+    "data-hub-match-merge-writer",
+    "data-hub-step-definition-reader",
+    "data-hub-step-definition-writer"
```

Comment:
*-writer roles inherit corresponding reader role. So can we only keep the writer roles?

Reply:
Good point, that's simpler - I'll make the change.